src/js/Framework/FormCollection.js

```diff
@@ -1,20 +1,23 @@
 import Sortable from 'sortablejs'
 
 const FormCollection = function (element) {
-  const addButton = element.querySelector('[data-role="collection-add-button"]')
+  this.addButton = element.querySelector('[data-role="collection-add-button"]')
```

Comment:
Is this best practice? In the `addItem`-method you also have the element available.